library/spdm_common_lib/libspdm_com_opaque_data.c

```diff
@@ -220,6 +219,13 @@ libspdm_process_opaque_data_supported_version_data(libspdm_context_t *spdm_conte
     spdm_version_number_t common_version;
     uint8_t version_count;
     bool result;
+    uint8_t element_index;
+    uint8_t element_num;
+    size_t current_element_len;
+    size_t total_element_len;
+    size_t total_opaque_len;
+
+    total_element_len = 0;
```

Comment:
I think we also need update libspdm_process_opaque_data_version_selection_data().

I dont see that update for that function.

Reply:
I have updated the libspdm_process_opaque_data_version_selection_data.
![image](https://user-images.githubusercontent.com/87060980/173012999-13ce9e24-0e58-4e42-9937-10cad9b11e93.png)